torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -360,6 +359,77 @@ def load_arg(a):
             env[node.name] = folded_graph.node_copy(node, load_arg)
     return QuantizedGraphModule(quantized_root, folded_graph, quantized_root.preserved_attr_names)
 
+def _get_module(node: Node, modules: List[torch.nn.Module]):
+    """
+    Return the torch.nn.Module that corresponds to the specified node's target.
+    If no such node exists, return None.
+    """
+    if node.op == "call_module" and str(node.target) in modules:
+        return modules[str(node.target)]
+    else:
+        return None
+
+def _match_static_pattern(
+        node: Node,
+        modules: Dict[str, torch.nn.Module],
+        qconfig_map: Dict[str, QConfigAny],
+        matching_ops: List[Callable] = None,
+        skip_dequantize_matching: bool = False):
+    """
+    Match the pattern (dequantize - ref node - quantize) with the node provided.
+
+    If there is a match, return a 3-tuple of:
+      1) q_node: the quantize node
+      2) relu_node: a relu module or functional wrapping the ref_node, and
+      3) ref_node: a reference module or functional to replace with the quantized counterpart
+
+    Otherwise, if there is no match, return a 3-tuple of (None, None, None)
+
+    If `matching_ops` is provided, then assume we are matching against the provided list of functional ops.
+    Otherwise, assume we are matching against `STATIC_LOWER_MODULE_MAP` and `STATIC_LOWER_FUSED_MODULE_MAP`.
+    """
+    skip_lowering_value = (None, None, None)
+    match_module = matching_ops is None
+
+    # Match quantize node
+    if node.op != "call_function" or node.target != torch.quantize_per_tensor:
+        return skip_lowering_value
+    q_node = node
+    ref_node = q_node.args[0]
+
+    # Handle cases where the node is wrapped in a ReLU
+    if (ref_node.op == "call_function" and ref_node.target in (F.relu, torch.relu)) or\
+            (ref_node.op == "call_module" and type(_get_module(ref_node, modules)) == nn.ReLU):
+        relu_node = ref_node
+        ref_node = relu_node.args[0]
+    else:
+        relu_node = None
+    if should_skip_lowering(ref_node, qconfig_map):
+        return skip_lowering_value
+
+    # Match reference module or functional
+    if match_module:
+        if ref_node.op != "call_module":
+            return skip_lowering_value
+        ref_class = type(_get_module(ref_node, modules))
```

Comment:
module = modules[str(node.target)]

get_target_type(object):
# object can be nn.Module instance or functional/torch op
if isistance(object, nn.Module): return type(object0)
else: return object